modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc

```diff
@@ -0,0 +1,237 @@
+
+[source]
+----
+xref:anchor_name_attribute-name[]
+----
+
+Example:
+
+[source]
+----
+For details, see xref:module_A_being_reused_assembly-1-name[].
+----
+====
+
+== Practical Example 1: Reusing Modules in Multiple Assemblies
+
+In a user guide, you want to reuse the "Creating Assets" procedure module and the "Projects" concept module in both "Chapter 3: Assets" and "Chapter 4: Projects" assemblies.
+
+The module files contain the following content:
+
+.projects-con.adoc
+[source]
+----
+// Module included in the following assemblies: "assembly_projects.adoc", "assembly_assets.adoc"
+
+[#projects_con_{context}]
+= Projects
+----
+
+.assets-creating-proc.adoc
+[source]
+----
+// Module included in the following assemblies: "assembly_projects.adoc", "assembly_assets.adoc"
+
+[#assets_creating_proc_{context}]
+= Creating Assets
+----
+
+The assembly files contain the following content:
+
+.assembly_projects.adoc
+[source]
+----
+include::organizational-unit-con.adoc
+
+include::repository-con.adoc
+
+:context: assembly-projects
+include::projects-con.adoc
+
+include::organizational-unit-con.adoc
+
+include::packages-create-proc.adoc
+
+:context: assembly-projects
+include::assets-creating-proc.adoc
+
+include::dependencies-add-proc.adoc
+----
+
+.assembly_assets.adoc
+[source]
+----
+include::assets-business-processes-con.adoc
+
+:context: assembly-assets
+include::projects-con.adoc
+
+include::projects-types-ref.adoc
+
+include::assets-packages-gloss.adoc
+
+:context: assembly-assets
+include::assets-creating-proc.adoc
+----
+
+Then for cross-references to the reused modules, you specify which context (assembly) you want to link to. For example, you can link to the "Projects" module as it appears either in the "Chapter 3: Assets" assembly or in the "Chapter 4: Projects" assembly:
+
+[source]
+----
+For details, see xref:projects_con_assembly-assets[].
+
+or
+
+For details, see xref:projects_con_assembly-projects[].
+
+----
+
+== Practical Example 2: Reusing a Module in a Single Assembly
+
+In a user guide, you want to reuse the "Projects" concept module two times in the same "Chapter 4: Projects" assembly.
+
+The module file contains the following content:
+
+.projects-con.adoc
+[source]
+----
+[#projects_con_{context}]
+= Projects
+----
+
+The assembly file contains the following content:
+
+.assembly_projects.adoc
+[source]
+----
+:context: intro
+include::projects-con.adoc
+
+include::organizational-unit-con.adoc
+
+include::asset-types-ref.adoc
+
+:context: definitions
+include::projects-con.adoc
+
+include::dependencies-con.adoc
+----
+
+Then for cross-references to the reused module, you specify which context (section) you want to link to. For example, you can link to the "Projects" module as it appears either in the introduction or in the definitions section:
+
+[source]
+----
+For details, see xref:projects_con_intro[].
+
+or
+
+For details, see xref:projects_con_definitions[].
+----
+
+== Additional Resources
```

Comment:
In the `assembly_writing-mod-docs.adoc`, this section is immediately followed by the assembly's _Additional Resources_ section, which looks strange. Can this information be put somewhere else?

Reply:
I'll just remove that section. I see little value in it, IMO, at least not as much as the Additional Resources stacked right before it. Plus it's already referred to in guidelines-assemblies.